worker/src/RTC/Transport.cpp

```diff
@@ -2260,6 +2260,7 @@ namespace RTC
 				auto* consumer = it->second;
 				auto bweType   = this->tccClient->GetBweType();
 
+				// NOLINTNEXTLINE(bugprone-too-small-loop-variable)
```

Comment:
`1u` in the second statement is considered of a bigger unit than the `uint8_t` defined in the first statement. Thus, the `NOLINTNEXTLINE` statement.